tasks/main.yml

```diff
@@ -2,6 +2,9 @@
 - name: Include tasks for sanity checks
   ansible.builtin.import_tasks: "sanity-check.yml"
 
+- name: Create Swarm Client user/group
+  ansible.builtin.import_tasks: "add-jenkins-user.yml"
```

Comment:
In 787ddc5a24c6cc21df5acb5b48cd689b3116044c, this is done a bit too early. We shouldn't add a Jenkins user if we don't even know the Swarm Client version to install.

Reply:
I guess it is possible for `get-swarm-client-version.yml` to bail out with an error. Fixed in f2606fe.